src/Illuminate/Database/Query/Builder.php

```diff
@@ -3840,7 +3840,7 @@ public function insertOrIgnoreUsing(array $columns, $query)
     /**
      * Update records in the database.
      *
-     * @param  array  $values
+     * @param  array<string, mixed>  $values
```

Comment:
`mixed` might be improved:
```suggestion
     * @param  array<string, \Illuminate\Database\Query\Builder|\Illuminate\Support\Collection|\UnitEnum|mixed>  $values
```

Reply:
Hey, that feels like a more involved change than I was planning to make here (although I'm happy to implement it if desired by the maintainers).

I'm also not sure of the value of adding other types, in a union with `mixed`? With it being the top type, I believe `X|mixed` is equivalent to `mixed` (for any `X`).

Reply:
Yeah, I would like to get rid of the mixed altogether but at a quick glance, I didn't know what were possible types for the `default` `match` case. I just thought: What good is it to retroactively add doc block types and then still end up with `mixed`? That might not be your fault, but I would be trying to avoid that to achieve an actually improved typing

Reply:
I think that mixed is enough here for now.